prices/providers/google.yml

```diff
@@ -8,88 +8,11 @@ pricing_urls:
 api_pattern: 'https://(.*\.)?googleapis\.com'
 model_match:
   contains: gemini
+provider_match:
+  or:
+    - contains: google
+    - equals: gemini
 models:
-  - id: claude-3-5-haiku
```

Comment:
Why have these been removed? even if they should be removed, it should be in a separate PR.